server/medication/collection.ts

```diff
@@ -65,7 +65,7 @@ export default class MedicalContactCollection {
    */
   static async deleteOne(medicationId: string): Promise<void> {
     const medicalContact = await MedicationModel.findOne({ _id: medicationId });
```

Comment:
[nit] should this say medication instead? Just for consistency

Reply:
hahaha yep.... as you can see I was copying your code lol